openpype/hosts/maya/plugins/publish/extract_multiverse_usd.py

```diff
@@ -129,13 +131,19 @@ def parse_overrides(self, instance, options):
 
         return options
 
+    def get_file_format(self, instance):
```

Comment:
You call `get_file_format` but it does not return a thing. Instead it is changing attribute of object so if instance does have empty fileFormats it will use `scene_type` of previous instance.

Also not sure what the code should do? If it would be rewritten to simplified version:
```
instance_file_format = ["usd", "usda"]
file_formats = ["usd", "usda", "usdz"]
some_range = range(len(file_formats)) # that's the same as [0, 1, 2]

# This will never be true as the list with file formats won't be in iter of itegers
if instance_file_format in some_range:
    # You change attribut on the class instead of returning it
    # + it would crash if it would get here because it's using
    #    a list to get item from list (TypeError)
    self.scene_type = file_formats[instance_file_format]
```

It kinda looks like you wanted to use `for` instead of `if`? But that's anti-pythonic way of looping through list. Also only last item would be used in that case it would be better to use:
```
# Backwards compatibility for instances without fileFormat
file_format = instance.data.get("fileFormat")
# Check if is set
if file_format:
    # return last item of the list
    return file_format[-1]
# Return default scene type
return self.default_file_format
```

Reply:
Why would be `instance.data["fileFormat"]` be a list or tuple in the first place - that sounds confusing?

Why can't it be this?
```python
file_format = instance.data.get("fileFormat",
                                self.default_file_format)
```

Or even a hardcoded backwards compatible default:
```python
file_format = instance.data.get("fileFormat", 
                                # Backwards compatibility
                                "usd")
```

With this simplification it doesn't need its own `get_file_format` method. Nonetheless, like mentioned before.. please return the value instead of storing it on the class if you do end up using a defined method for this.

Reply:
Sorry for the delay. For simplicity, I did your suggestion of using that `get` method.